packages/vue/src/components/status-select-box.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <dx-select-box
+    label="Status"
+    :value="modelValue"
+    @value-changed="emitChangedValue"
+    :items="items"
+    :read-only="readOnly"
+    :styling-mode="stylingMode"
+    :label-mode="labelMode"
+    width="100%"
+    field-template="field"
+  >
+    <template #field="{data}">
+      <div class="status-editor-field">
+        <contact-status
+          class="status-indicator"
+          :show-text="false"
+          :value="data"
+        />
+        <dx-text-box
+          class="status-{{data.toLowerCase()}}"
+          :hover-state-enabled="false"
+          :input-attr="{
+            // eslint-disable-next-line max-len
+            class: `status-editor-input status-input contact-status status-${data?.toLowerCase()}`}"
```

Comment:
Could we move class names to new lines? And get rid of eslint comment after it

Reply:
fix